pandas/tests/frame/methods/test_reset_index.py

```diff
@@ -671,3 +683,15 @@ def test_reset_index_multiindex_nat():
         index=pd.DatetimeIndex(["2015-07-01", "2015-07-02", "NaT"], name="tstamp"),
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_inplace_deprecation_warning():
```

Comment:
I think this can be removed since this is covered below?

Reply:
I've put the test for `inplace=False` in here too now as I think it fits better here than in the test which uses `"True"` as a string rather than `True` - reckon this is OK?

Reply:
Yes of course.